featurebyte/worker/task/feature_job_setting_analysis_backtest.py

```diff
@@ -94,9 +94,9 @@ async def execute(self, payload: FeatureJobSettingAnalysisBackTestTaskPayload) -
         # run backtest
         await self.task_progress_updater.update_progress(percent=5, message="Running Analysis")
         feature_job_setting = FeatureJobSetting(
-            frequency=payload.period,
+            period=payload.period,
```

Comment:
@gee-senbong I think we don't need to modify `recommended_feature_job_setting` anymore

Reply:
`FeatureJobSetting` expects `period` and `offset` parameters in https://github.com/featurebyte/freeware/blob/main/featurebyte_freeware/feature_job_analysis/schema.py#L352-L353.

Reply:
I am referring to the `from_dict` line above this change. Support for the old parameters `frequency` and `job_time_modulo_frequency` has been added here:
https://github.com/featurebyte/freeware/blob/main/featurebyte_freeware/feature_job_analysis/schema.py#L377